SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+            supplementary statement.
+        </p>
+        <p class="govuk-body">
+            You have the right to request that your personal data are erased if there is no longer a justification for
+            them to be processed.
+        </p>
+        <p class="govuk-body">
+            You have the right in certain circumstances (for example, where accuracy is contested) to request that the
+            processing of your personal data is restricted.
+        </p>
+        <p class="govuk-body">
+            You have the right to object to the processing of your personal data where it is processed for direct
+            marketing purposes.
+        </p>
+        <p class="govuk-body">
+            You have the right to object to the processing of your personal data.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            International Transfers
+        </h2>
+        <p class="govuk-body">
+            As your personal data is stored on our IT infrastructure and shared with our data processors Microsoft and
+            Amazon Web Services, it may be transferred and stored securely in the UK and European Economic Area. Where
+            it is the case that your personal data is stored outside the UK and EEA it will be subject to equivalent 
+            legal protection through the use of Model Contract Clauses.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Complaints
+        </h2>
+        <p class="govuk-body">
+            If you consider that your personal data has been misused or mishandled, you may make a complaint to the
+            Information Commissioner, who is an independent regulator.
+        </p>
+        <p class="govuk-body">
+            The Information Commissioner can be contacted at:
+        </p>
+        
+        <div class="govuk-inset-text">
+            <p class="govuk-body">
+                Information Commissioner's Office<br>
+                Wycliffe House<br>
+                Water Lane<br>
+                Wilmslow<br>
+                Cheshire<br>
+                SK9 5AF<br>
+            </p>
+            <ul class="govuk-list">
+                <li>Telephone: 0303 123 1113</li>
+                <li>Monday to Friday, 9am to 5pm (except public holidays)</li>
+            </ul>
+            <p class="govuk-body">
+                Email: <a class="govuk-link" href="mailto: casework@ico.org.uk">casework@ico.org.uk</a>
+            </p>
+        </div>
+        
+        <p class="govuk-body">
+            Any complaint to the Information Commissioner is without prejudice to your right to seek redress through
+            the courts. 
+        </p>
+        
+        <h2 class="govuk-heading-s">Contact Details</h2>
+        <p class="govuk-body">
+            The data controller for your personal data is the Department for Business, Energy &amp; Industrial Strategy
```

Comment:
Is there a reason to use `&amp` instead of `&`?

Reply:
`&` is an escape character in html so it won't show up `&amp;` is the sequence for `&` (like `&gt;` will get you a `>`)